TestAssets/TestProjects/FSharpTestProjects/TestLibrary/project.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
AFAIK plan is to move the whole shebang to a new repo, @enricosada any update on that repo coming up?